release-1.8/release-1.8.md

```diff
@@ -0,0 +1,79 @@
+# Proposed timeline for 1.8
+We will begin the 1.8 release the first week of July 2017.
+
+The proposal below is similar in layout to the 1.7 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.8 Release Schedule
+- July 5 (Weds) coding start (7w)
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1 (Tues): Feature Freeze
+- Aug 9 (Weds): v1.8.0-alpha.3
+- Aug 23 (Weds): v1.8.0-alpha.4
+- Sept 1 (Fri): Code Freeze 
+- Sept 6 (Weds): v1.8.0-beta.0, v1.9.0-alpha.0
+- Sept 13 (Weds): v1.8.0-beta.1
+- Sept 18th (Mon): v1.9.0-alpha.1
+- Sept 20th (Weds): v1.8.release-candidate
+- Sept 27th (Weds):  v1.8.0 (release!)
+
+## 1.8 Details
+
+### July 5 - Sept 1
+- 8 week coding period
+- Release 1.8 alphas every 2 weeks
```

Comment:
> By "all release tests", do you mean all the jobs displayed on the release-master-blocking testgrid dashboard? AFAIK this what kubernetes/release uses

I think this makes sense at least until we can get better ownership of non-blocking suites. If we do include others at this point we should explicitly list them since the status quo is currently having some bad jobs. This would make sense for important suites that are likely non-blocking for test duration reasons like serial and soak.

> By "be passing" do you mean their most-recent "Overall" results are green? Some noise I've heard in the past: human explanations for why we could ignore red results in some mostly green suites; a practice of waiting for "3 in a row" prior to calling a final release good

The "3 in a row" is the standard I've seen for the past few releases but it's generally been up to human discretion. We should decide on an exact criteria and ideally have it machine enforced.